scripts/create_addon

```diff
@@ -91,6 +91,13 @@ pack_addon() {
   if [ ! -z "$PKG_ADDON_NAME" ] ; then
     ADDON_NAME="$PKG_ADDON_NAME"
   fi
+  for f in $PKG_DIR/repo/screenshot-*.{jpg,png}; do if [ -f $f ];
```

Comment:
I'd definitely move the `if [ ]; then` to its own line, to stay consistent with existing code/style.